pkg/hostpath/hostpath.go

```diff
@@ -347,6 +363,19 @@ func shareUpdateRanger(key, value interface{}) bool {
 			klog.Warningf("share %s vol %s target path %s delete error %s",
 				shareId, volID, oldTargetPath, err.Error())
 		}
+		if hpv.IsReadOnly() {
+			tp := hpv.GetTargetPath()
+			empty, err := isDirEmpty(tp)
+			errStr := ""
+			if err != nil {
+				errStr = err.Error()
+			}
+			klog.V(0).Infof("GGM shareUpdateRanger kubelet dir %s empty %v err %s", tp, empty, errStr)
```

Comment:
this `hpv.IsReadOnly` if block is for https://github.com/openshift/csi-driver-projected-resource/pull/35/commits/5d7cba12da8bb202f22eee00ba7d7df69d14e482 which is a *possible* means for removing content

a) after the driver restarts
b) then permissions to the share are removed

I was able to verify this multiple times in my testing.  However, why I am able to *remove* content under the kubelet mount dir after a restart, but am unable to *update* or *change* content under the kubelet mount dir after a restart, is still a mystery to me.

Pending learning the precise *why*, I also wonder if this limited form of update support for read only volumes after driver restart will be worth the effort to maintain, document, explain, etc. moving forward will be worth it.